test/python/evaluation/statistics/test_raw_issue_encoding_decoding.py

```diff
@@ -0,0 +1,210 @@
+import json
+import textwrap
+from pathlib import Path
+
+import pytest
+from src.python.evaluation.statistics.common.raw_issue_encoder_decoder import RawIssueDecoder, RawIssueEncoder
+from src.python.review.inspectors.inspector_type import InspectorType
+from src.python.review.inspectors.issue import (
+    BaseIssue,
+    BoolExprLenIssue,
+    CodeIssue,
+    CohesionIssue,
+    CyclomaticComplexityIssue,
+    FuncLenIssue,
+    IssueType,
+    LineLenIssue,
+    MaintainabilityLackIssue,
+)
+
+ISSUE_AND_JSON_ISSUE = [
+    (
+        CodeIssue(
+            origin_class='SomeCodeIssueClass',
+            type=IssueType.CODE_STYLE,
+            description="Some description",
+            file_path=Path("/home/user/Desktop/some_file2.py"),
```

Comment:
What about this path? And all paths in this file? Can you create a const variable and use a simple name?